conda.rb

```diff
@@ -7,14 +7,50 @@
 class Conda
   include Singleton
 
-  attr_reader :main
-
   def initialize
-    @main = Conda::Channel.new("pkgs/main", "repo.anaconda.com")
+    @channels = {
+      "main" => Conda::Channel.new("main", "repo.anaconda.com/pkgs"),
+      "conda-forge" => Conda::Channel.new("conda-forge", "conda.anaconda.org"),
+    }
+  end
+
+  def packages(channel)
+    raise Sinatra::NotFound unless @channels.key?(channel)
+
+    @channels[channel].packages
+  end
+
+  def all_packages
+    all_packages = {}
+    @channels.each_value do |channel|
+      channel.only_one_version_packages.each do |package_name, package|
+        if all_packages.key?(package_name)
+          all_packages[package_name][:versions] = all_packages[package_name][:versions] + package[:versions]
+        else
+          all_packages[package_name] = package
+        end
+      end
+    end
+    all_packages
+  end
+
+  def package(channel, name)
+    packs = packages(channel)
+    raise Sinatra::NotFound unless packs.key?(name)
+
+    packs[name]
+  end
+
+  def find_package(name)
+    @channels.values.find { |channel| channel.packages.key?(name) }&.packages&.dig(name)
```

Comment:
Maybe we should throw the Sinatra::NotFound here to match package(channel, name) and then we don't have to worry 'bout it in the "controller"?